generator/csv/fields_easyrpg.csv

```diff
@@ -0,0 +1,3 @@
+#Structure,Field,Size Field?,Type,Index,Default Value,PersistIfDefault,Is2k3,Comment
+Save,easyrpg_data,f,SaveEasyRpgData,0xc8,,0,0,Additional save data written by EasyRPG Player
+SaveEasyRpgData,version,,Enum<EasyRpg_Version>,1,0,0,0,Savegame version
```

Comment:
Is this `enum` a leftover? Seems unused.

Reply:
I don't even know how I got this commited, I deleted the file before o.O

Reply:
oh this is in fields, not enum. good find!